README.md

```diff
@@ -316,6 +352,28 @@ running:
 
 ```sh
 $ uuid=$(rekor-cli search --artifact <(git rev-parse HEAD | tr -d '\n') | tail -n 1)
+$ rekor-cli get --uuid=$uuid --format=json | jq .
```

Comment:
I think we should also mention the UI:
- https://rekor.tlog.dev/?logIndex=2212633
- https://rekor.tlog.dev/?uuid=d0444ed9897f31fefc820ade9a706188a3bb030055421c91e64475a8c955ae2c
- https://rekor.tlog.dev/?email=billy@chainguard.dev
- https://rekor.tlog.dev/?commitSHA=[...]